test/support/builders/atom_watcher_event.js

```diff
@@ -27,11 +27,20 @@ module.exports = class AtomWatcherEventBuilder {
     if (old) {
       this._event = _.cloneDeep(old)
     } else {
+      const stats = _.merge(new fs.Stats(), {
+        ino: 1,
```

Comment:
I'm not sure to remember how inodes are used in the new watcher...
Should we ensure 2 generated events don't end up with the same inode by default?
(maybe it doesn't matter)

Reply:
Hum this could be a good practice.

Reply:
(not a blocker for this PR since previous implementation was always using the same inode too)